CONTRIBUTING.md

```diff
@@ -72,6 +72,12 @@ kind load docker-image quay.io/open-cluster-management/placement:latest --name {
 make test-e2e
 ```
 
+Run scalability testing if your PR has impact on the scalability
```

Comment:
is there any pre-requirement for this? Do we need a cluster installed already? Or is it possible to run  this test against a ocm that has been installed already/.

Reply:
Thanks for the reminder, yes, we need to deploy the placement controller before running this test case, added following message to remind the developer:

`Deploy the placement controller with corresponding steps from [PLACEMENT Doc](https://github.com/open-cluster-management-io/placement#deploy-the-placement-controller)`

BTW: `make test-e2e` and `make test-integration` share the same issue, I will update them too in another PR after this PR is merged.